### PR TITLE
refactor content

### PR DESCRIPTION
**Pull request by aweiteka**
First of a few dependent PRs to make the content more compelling and clear.

Signed-off-by: Aaron Weitekamp <aweiteka@redhat.com>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -35,8 +62,9 @@ The architectural components are as follows:
 - The CRI Image API is implemented using the `containers/image` library.
 - Networking for the pod is setup through use of [CNI](https://github.com/containernetworking/cni), so any CNI plugin can be used with CRI-O.
 
-CRI-O is made up of several components that are found in different GitHub repositories.
-Let’s take a look at the components.  
+## Components
+
+CRI-O is made up of several components that are found in different GitHub repositories.
```
Comment (jzb) [inline index.md]: Do we want to list the components?

File: index.md [hunk 2]
```diff
@@ -73,19 +101,7 @@ situations.
...
-
-1. Basic pod/container lifecycle, basic image pull (already works)
-1. Support for tty handling and state management
-1. Basic integration with kubelet once client side changes are ready
-1. Support for log management, networking integration using CNI, pluggable image/storage management
-1. Support for exec/attach
-1. Target fully automated kubernetes testing without failures
-
-
+<!--
```
Comment (jtligon) [inline index.md]: are we getting rid of the twitter feed? I love the feed!

Comment (aweiteka) [inline index.md]: I wasn't sure. I can pull it back in if it's "loved" :)

File: index.md (hunk 1 above)
Comment (aweiteka) [inline index.md]: The components are network, security, container images, etc.

**Comment by jtligon** [comment]
I’d prefer to explicitly link out to those other repos in a bulleted list.
Do we have the full list?

On July 11, 2017 at 3:22:32 PM, Aaron Weitekamp (notifications@github.com)
wrote:

*@aweiteka* commented on this pull request.
------------------------------

In index.md <https://github.com/cri-o/cri-o.io/pull/8#discussion_r126783983>
:

> @@ -35,8 +62,9 @@ The architectural components are as follows:
 - The CRI Image API is implemented using the `containers/image` library.
 - Networking for the pod is setup through use of
[CNI](https://github.com/containernetworking/cni), so any CNI plugin
can be used with CRI-O.

-CRI-O is made up of several components that are found in different
GitHub repositories.
-Let’s take a look at the components.
+## Components
+
+CRI-O is made up of several components that are found in different
GitHub repositories.

The components are network, security, container images, etc.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub
<https://github.com/cri-o/cri-o.io/pull/8#discussion_r126783983>, or mute
the thread

**Pull request merged**